server/controllers/annotation_controller.js

```diff
@@ -46,6 +46,7 @@ export const createAnnotation = (user, body, articleId) => {
       });
   } else {
```

Comment:
Should we also give each child annotation a ranges field, which would have the same value as its parent's? Or did you only want to store this for top-level annotations?

If we are never planning to access an annotation without also showing all of its ancestors along with the article text, then I guess it makes sense to only store the ranges in the top annotation.